src/content/blog/astro-2100.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: "Astro 2.10: Persistent State in View Transitions"
+description: "Persistent UI in View Transitions and our first look at Astro 3.0"
+publishDate: "August 3, 2023"
+authors:
+  - matthew
+coverImage: "/src/content/blog/_images/astro-2100/blog-hero-2_10-release.png"
+socialImage: "/src/content/blog/_images/astro-2100/og-image-2_10-release.png"
+lang: "en"
+---
+
+Astro 2.10 is out with improvements to our experimental View transitions support. Additionally we've released our first early beta of Astro 3.0. Read on to learn more!
+
+
+- **[Persistent UI in View transitions](#view-transitions-experimental)**
+- **[First Look at Astro 3.0](#astro-3)**
+
+If you already have Astro installed, you can upgrade it to 2.10 by running the upgrade command in your project (using your package manager of choice):
+
+```
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+While you're at it, upgrade any `@astrojs/*` integrations and adapters you have installed, too!
+
+## Persistent UI in View Transitions
+
+Our first release of the `<ViewTransition />` router brought smooth transitions between pages, but still caused any stateful islands or elements to be rerendered.
```

Comment:
Comma in compound predicate isn't necessary, also missing hyphen in `rerendered`
```suggestion
Our first release of the `<ViewTransition />` router brought smooth transitions between pages but still caused any stateful islands or elements to be re-rendered.
```